src/modules/saveTodo.js

```diff
@@ -0,0 +1,10 @@
+const saveTodo = (newTodo) => {
+  let todoList = [];
+  if (localStorage.getItem('todo')) {
+    todoList = JSON.parse(localStorage.getItem('todo'));
+  }
+  todoList.push(newTodo);
+  localStorage.setItem('todo', JSON.stringify(todoList));
+};
```

Comment:
- [ ] Good job @girma336, you have done most of the requirements, The property `index` should set to the value of the new array length, The first task should have `index=1` (i.e. if a 5th task is added to the list, the index of that task should equal to 5).
![Capture](https://user-images.githubusercontent.com/69077061/201047409-b52c96b9-6209-4233-ae77-240bbce3f0a2.PNG)
- [Optional] Your page reloads on every input, It would be great to not have to reload on every input.